src/pages/Home/Home.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
+import { Grid, Container } from '@mui/material';
+
+import Header from '@/components/Home/HomeHeader';
+import HeaderBackground from '@/components/Home/HomeHeaderBackground';
+import HomeRoomList from '@/components/Home/HomeRoomList';
+import HomeFooter from '@/components/Home/HomeFooter';
```

Comment:
For the naming of these components and file names, I would recommend keeping it simple and remove all the 'Home' prefix (eg. `RoomList` `Footer`). it's already quite clear that this component will be used in the home page, since it's located in the `components/home/`. 

Also, keep the naming consistent! Eg. `import Header from '.../HomeHeader'` is not consistent